src/app/errors/components/PageNotFound.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
+import { FormattedMessage } from 'react-intl';
 
 export default () => {
-  return <h1>page not found</h1>;
+  return <h1><FormattedMessage id='i18n:404'/></h1>;
```

Comment:
can we use this syntax to avoid the extraneous spans?

```
<FormattedMessage id="title">
    {(txt) => (
        <H1>
            {txt}
        </H1>
    )}
</FormattedMessage>
```

from https://github.com/yahoo/react-intl/wiki/Components#formattedmessage